action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
   general-settings-json: #id of input
     description: 'General configuration settings using dictionary syntax - Key Value pairs'
     required: false
+  mask-inputs: 

```

Comment:
close the name and description with PMs

Reply:
@N-Usha Can you please review this?

Reply:
name looks ok.. But would this apply to all inputs including general-settings? Else needs to be called out in description.

Reply:
@N-Usha Currently in v1 general-settings is not masked. But in master it is masked. Hence in this change with mask-Inputs, it will be masked.
if mask-inputs is set as "false" by user, do not mask any settings
if mask-inputs is not set by user, default is true and all 3 settings will be masked (including general-settings)
if mask-inputs is set to anything other than "false", all 3 inputs will be masked.
Please let me know should i update this in action.yml

Reply:
got it. Hope the value 'false' specified is not case sensitive.. If not, should be called out in description again.

Reply:
@N-Usha It is not case sensitive.